packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -57,6 +43,35 @@ module.exports = class SingleSpaReactGenerator extends Generator {
         ])
       ).typescript;
     }
+
+    const packageJsonTemplate = await fs.readFile(
+      this.templatePath("package.json"),
+      { encoding: "utf-8" }
+    );
+    const packageJsonStr = ejs.render(packageJsonTemplate, {
+      packageManager: this.packageManager,
+      typescript: this.typescript,
+    });
+
+    const packageJson = JSON.parse(packageJsonStr);
+
+    if (this.typescript) {
+      // Will be added as a dependency via ts-package.json
+      delete packageJson.devDependencies["@types/jest"];
```

Comment:
Previously, `@types/jest` was being added as both a dependency and a devDependency

Reply:
Not sure I understand. Shouldn't this only be installed if TS is present, and only as a normal dependency? Why is it in `devDependencies` in the first place?

Unless, is the idea that this generator should update previously-generated projects?

Reply:
It is there for non typescript projects simply so VS Code doesn't freak out or try incorrect autocompletions for `describe()` and `it()`. Non typescript projects can benefit from typescript simply by having their IDE look inside of their node_modules